### ISSUE TITLE
Error in pretty print of sqrt(), conjugate(Complex)  

### ISSUE THREAD

**Issue by VinaySingh035** [sympy/sympy#10107]
when called for real variable:
```
In [15]: from sympy import *

In [16]: x=symbols('x', real=True)

In [17]: print(conjugate(x))
x

In [18]: pprint(conjugate(x))
x
```
and for complex:
```
In [19]: x=symbols('x', complex=True)

In [20]: pprint(conjugate(x))
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-20-3caf28518f4e> in <module>()
----> 1 pprint(conjugate(x))

/home/vinay/sympy/sympy/printing/pretty/pretty.pyc in pretty_print(expr, **settings)
   2031 
   2032     """
-> 2033     print(pretty(expr, **settings))
   2034 
   2035 pprint = pretty_print

/home/vinay/sympy/sympy/printing/pretty/pretty.pyc in pretty(expr, **settings)
   1998 
   1999     try:
-> 2000         return pp.doprint(expr)
   2001     finally:
   2002         pretty_use_unicode(uflag)

/home/vinay/sympy/sympy/printing/pretty/pretty.pyc in doprint(self, expr)
     56 
     57     def doprint(self, expr):
---> 58         return self._print(expr).render(**self._settings)
     59 
     60     # empty op so _print(stringPict) returns the same

/home/vinay/sympy/sympy/printing/printer.pyc in _print(self, expr, *args, **kwargs)
    255                 printmethod = '_print_' + cls.__name__
    256                 if hasattr(self, printmethod):
--> 257                     return getattr(self, printmethod)(expr, *args, **kwargs)
    258 
    259             # Unknown object, fall back to the emptyPrinter.

/home/vinay/sympy/sympy/printing/pretty/pretty.pyc in _print_conjugate(self, e)
    233     def _print_conjugate(self, e):
    234         pform = self._print(e.args[0])
--> 235         return prettyForm( *pform.above( hobj('_', pform.width())) )
    236 
    237     def _print_Abs(self, e):

/home/vinay/sympy/sympy/printing/pretty/stringpict.pyc in above(self, *args)
    172         Baseline is baseline of bottom picture.
    173         """
--> 174         string, baseline = stringPict.stack(*(args + (self,)))
    175         baseline = len(string.splitlines()) - self.height() + self.baseline
    176         return string, baseline

/home/vinay/sympy/sympy/printing/pretty/stringpict.pyc in stack(*args)
    145             newPicture.extend(obj.picture)
    146         newPicture = [line.center(newWidth) for line in newPicture]
--> 147         newBaseline = objects[0].height() + objects[1].baseline
    148         return '\n'.join(newPicture), newBaseline
    149 

TypeError: 'int' object is not callable
```

similar for sqrt() function.
```
In [21]: from sympy import *

In [22]: x=symbols('x')

In [23]: pprint(sqrt(x))
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-23-379750182b76> in <module>()
----> 1 pprint(sqrt(x))

/home/vinay/sympy/sympy/printing/pretty/pretty.pyc in pretty_print(expr, **settings)
   2031 
   2032     """
-> 2033     print(pretty(expr, **settings))
   2034 
   2035 pprint = pretty_print

/home/vinay/sympy/sympy/printing/pretty/pretty.pyc in pretty(expr, **settings)
   1998 
   1999     try:
-> 2000         return pp.doprint(expr)
   2001     finally:
   2002         pretty_use_unicode(uflag)

/home/vinay/sympy/sympy/printing/pretty/pretty.pyc in doprint(self, expr)
     56 
     57     def doprint(self, expr):
---> 58         return self._print(expr).render(**self._settings)
     59 
     60     # empty op so _print(stringPict) returns the same

/home/vinay/sympy/sympy/printing/printer.pyc in _print(self, expr, *args, **kwargs)
    255                 printmethod = '_print_' + cls.__name__
    256                 if hasattr(self, printmethod):
--> 257                     return getattr(self, printmethod)(expr, *args, **kwargs)
    258 
    259             # Unknown object, fall back to the emptyPrinter.

/home/vinay/sympy/sympy/printing/pretty/pretty.pyc in _print_Pow(self, power)
   1382             n, d = fraction(e)
   1383             if n is S.One and d.is_Atom and not e.is_Integer:
-> 1384                 return self._print_nth_root(b, e)
   1385             if e.is_Rational and e < 0:
   1386                 return prettyForm("1")/self._print(Pow(b, -e, evaluate=False))

/home/vinay/sympy/sympy/printing/pretty/pretty.pyc in _print_nth_root(self, base, expt)
   1334         # In very simple cases, use a single-char root sign
   1335         if (self._settings['use_unicode_sqrt_char'] and self._use_unicode
-> 1336             and expt is S.Half and bpretty.height() == 1
   1337             and (bpretty.width() == 1
   1338                  or (base.is_Integer and base.is_nonnegative))):

TypeError: 'int' object is not callable
```

**Comment by thisch** [comment]
I can't reproduce your issue. Which version of sympy are you using?

**Comment by manipalsingh013** [comment]
I'm also not getting any error, even on live sympy
![screenshot from 2015-11-04 03 53 20](https://cloud.githubusercontent.com/assets/13970876/10924615/f0ec9aa2-82ad-11e5-90b2-921490d7c250.png)

**Comment by VinaySingh035** [comment]
Mine veersion of sympy is "0.7.7.dev".  with python2.7.6.

giving different error when using the installed sympy.
```
In [4]: from sympy import *

In [5]: x=symbols('x')

In [6]: pprint(sqrt(x))
---------------------------------------------------------------------------
UnicodeEncodeError                        Traceback (most recent call last)
<ipython-input-6-379750182b76> in <module>()
----> 1 pprint(sqrt(x))

/usr/local/lib/python2.7/dist-packages/sympy-0.7.7.dev-py2.7.egg/sympy/printing/pretty/pretty.py in pretty_print(expr, **settings)
   2031 
   2032     """
-> 2033     print(pretty(expr, **settings))
   2034 
   2035 pprint = pretty_print

UnicodeEncodeError: 'ascii' codec can't encode character u'\u221a' in position 0: ordinal not in range(128)
```

the error above mentioned is the one I got when used the cloned sympy.
and this is the error when I used the sympy I installed via `sudo apt-get`.

**Comment by thisch** [comment]
I guess it is not using the version you installed via apt-get, as sympy is used from ``/usr/local``. You probably also installed your cloned repo to ``/usr/local``. Try to remove this folder from ``/usr/local`` and try it  again.

**Comment by asmeurer** [comment]
Are you on the `master` branch? I also don't see this error.

**Comment by VinaySingh035** [comment]
I have no idea as I an on Master branch or not.
I tried too hard to find the same error on my friends systems, but is unable to produce on anyone's system.

The Initial Error is found on the Sympy cloned.
` int object is not callable.`

and the `UnicodeEncodeError` is found when I used in the system's sympy. which I have installed using `sudo apt-get install`.
I think there is some problem with my systems sympy version.
as it says 0.7.7.dev.

help me to understand it.
